MerlinAU.sh

```diff
@@ -4142,7 +4143,13 @@ _EntwareServicesHandler_()
        done
        return 0
    }
-   
+
+   case "$1" in
```

Comment:
This is already fixed :)

Reply:
Merged and removed the duplicate here: https://github.com/ExtremeFiretop/MerlinAutoUpdate-Router/commit/bad5daf5381a092480130b25ecff39defff520ec